src/App.js

```diff
@@ -1,10 +1,23 @@
-
+import Header from './components/Header';
+import Navbar from './components/Navbar';
+import Reviews from './components/Review';
+import Comments from './components/Comments';
+import Users from './components/User';
+import Home from './components/Home';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
 
 function App() {
   return (
-    <div className="App">
-        
+    <div className='App'>
+      <Header />
+      <Navbar />
+      <Routes>
+        <Route path='/' element={<Home />}></Route>
+        <Route path='/review' element={<Reviews />}></Route>
+        <Route path='/review' element={<Comments />}></Route>
```

Comment:
You have two routes with the same path - the one on line 18's never going to happen
I'd also suggest calling it `/reviews` rather than `/review`